apps/frontend/app/(website)/studio/src/hooks/useShareLink.ts

```diff
@@ -55,26 +51,13 @@ export const useShareLink = () => {
   // 	}
   // };
 
-  const getURL = async (): Promise<URL | null> => {
+  const getURL = (): URL | null => {
     try {
       if (internalContent === null) {
         throw new Error("codemod content not found");
       }
 
       const codemodName = "untitled";
-      //   if (token !== null) {
```

Comment:
why we removed codemod name feature?

Reply:
this is for codemod sharing feature in studio, and the name field isn't currently consumed by anywhere by the receiver of the link. So it's overengineering.